src/main/java/uk/gov/hmcts/divorce/caseworker/event/CaseworkerOfflineDocumentVerified.java

```diff
@@ -203,9 +204,14 @@ public AboutToStartOrSubmitResponse<CaseData, State> aboutToSubmit(CaseDetails<C
                 caseData.getApplicant2().setOffline(YES);
             }
 
+            var isJudicialSeparation = caseData.getIsJudicialSeparation();
+            var state = isJudicialSeparation != null && isJudicialSeparation.equals(YES)
```

Comment:
why not make isJudicialSeparation YesNo type, then just put YES.equals(isJudicialSeparation) - no need for the null check

Reply:
Thanks Sanjay. I've implemented the above suggestion.